### PR TITLE
Removed cards from index.html, added new functionality to create high…

### PR DESCRIPTION
**Pull request by marcusmoller97**
Removed all elements with the class "card" in it from html to instead add the cards dynamically from the api instead.
Added some initial working code in challenges_rating_sort.js for sorting and adding of elements.

### CODE REVIEW

**Comment by Klumpish** [comment]
adding things to index.html is the wrong file. but other then that good job

File: index.html
Comment (Klumpish) [inline index.html:1]: seems to be wrong file, we should add this to challanges.html

**Review by Klumpish — CHANGES_REQUESTED** [review]
change the code from index.html to challanges.html

**Comment by jennymakki** [comment]
Looks good!

**Pull request closed without merging**